examples/pandas_dask_example.py

```diff
@@ -0,0 +1,77 @@
+import dask.datasets
```

Comment:
Wdyt @vijaykiran @m1n0  about adding an `examples` folder to the main directory to keep python API examples? We can eventually add other data sources like spark for the contributors.